.github/workflows/browserstack.yml

```diff
@@ -0,0 +1,64 @@
+name: Browserstack
+
+on:
+  push:
+    branches:
+      - migrate-test
```

Comment:
Just a comment to not forget to change it back to `main` before merging (& also `3.x-stable`? Or is that happening separately?)

Reply:
3.x-stable will be another PR. Since most of the test code runs on Node, I'm hoping it won't be too different.